core/container/src/test/java/org/wildfly/swarm/internal/FileSystemUtilsTest.java

```diff
@@ -0,0 +1,30 @@
+package org.wildfly.swarm.internal;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Optional;
+
+/**
+ * Created by hbraun on 22.08.17.
+ */
+public class FileSystemUtilsTest {
+
+    @Test
+    public void testMavenArgsPresent() {
+        String cmd = "clean -f pom.xml install";
```

Comment:
Should this test actually use a name other than `pom.xml` to ensure the code is actually taking any name from -f and not just ending up with pom.xml by default?